### PR TITLE
Feat: add filters

### PR DESCRIPTION
**Pull request by farhan-sauce**
- Add search input
- Add filters by type
- adjust filtersContainer style.

Screen Shot 2019-11-11 at 6 56 17 PM

### CODE REVIEW

File: src/Components/ImportHAR.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDropzone } from 'react-dropzone';
 import { Button } from 'react-bootstrap';
+
 import { useNetwork } from '../state/network/provider';
```
Comment (seanenright) [inline src/Components/ImportHAR.jsx]: can Styles and useNetwork be grouped?

File: src/Containers/FilterContainer.jsx [hunk 2]
```diff
@@ -1,10 +1,43 @@
 import React from 'react';
+import {
+  Row, Col, ButtonGroup, Button,
+} from 'react-bootstrap';
+
 import ImportHar from '../Components/ImportHAR';
+import Search from '../Components/Filters/Search';
```
Comment (seanenright) [inline src/Containers/FilterContainer.jsx]: just repeating my file path comment from the other PR 😸

File: src/Containers/NetworkTableContainer.jsx [hunk 3]
```diff
@@ -6,23 +6,34 @@ import NetworkTableRow from '../Components/NetworkTable/NetworkTableRow';
...
+      </section>
+    );
+  }
+
+  return (
+    <section className={Styles['table-container']}>
+      <Table striped bordered hover size="sm">
+        <NetworkTableHeader maxTime={totalNetworkTime} />
+        <tbody className={Styles['table-content']}>
+          {Array.from(data).map(rowInfo => (
```
Comment (seanenright) [inline src/Containers/NetworkTableContainer.jsx]: `((rowInfo) => (...))`

File: src/state/network/provider.jsx [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import React, { useReducer, useContext, useMemo } from 'react';
 
+import { actionsWrapper } from '../../utils';
+
```
Comment (seanenright) [inline src/state/network/provider.jsx]: can these be grouped?

File: src/state/network/reducer.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import { Map, List } from 'immutable';
 
-import * as types from './types';
 import { filterData, sortBy, prepareViewerData } from '../../utils';
```
Comment (seanenright) [inline src/state/network/reducer.js]: same, can this be grouped?

File: src/Components/Filters/Search.jsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  InputGroup, DropdownButton, Dropdown, FormControl,
+} from 'react-bootstrap';
+
+import Styles from './Search.module.scss';
```
[the remark this answers is not in the document]
Comment (farhan-sauce) [reply src/Components/Filters/Search.jsx]: `create-react-app` strict rules 🤦‍♂ 
In next PR, anyway I need to customize webpack and eject `create-react-app`. I will fix it there.

**Pull request merged**